### ISSUE TITLE
Contract information not loading {"statusCode":500,"error":"Internal Server Error","message":"Not enough bytes for magic prefix"}

### ISSUE THREAD

**Issue by isopen** [catchain/tonscan#45]
### Bug Type

UX/Functional

### Reproduction steps

<img src="https://user-images.githubusercontent.com/7373292/191168257-112cb165-3bd5-441c-99b0-94f2aaef349f.png" width="70%" />

<img src="https://user-images.githubusercontent.com/7373292/191168412-50984a8e-6314-4c98-8c7f-a2edc3360b7c.png" width="70%" />

[vokoscreenNG-2022-09-20_11-39-14.webm](https://user-images.githubusercontent.com/7373292/191168988-6a4bc117-9d5d-4668-9f8d-85f4c8739393.webm)

### Actual result

Contract block not loading

### Expected result

The contract block is loading

### Suggested Severity

Critical

### Device

Desktop (please complete the following information):
- OS: Ubuntu 22
- Browser Google Chrome
- Version 105.0.5195.52

Basically, it's a server error. Most likely will work from any device.
An additional check is needed to see if the server's hard drive may be filled with logs.

**Comment by ingria** [comment]
Duplicate #41
